modules/statics/src/coins.ts

```diff
@@ -3168,6 +3168,86 @@ export const coins = CoinMap.fromCoins([
     UnderlyingAsset.STSOL,
     AccountCoin.DEFAULT_FEATURES
   ),
+  solToken(
+    'sol:psol',
+    'pSOL (Parrot SOL)',
+    9,
+    '9EaLkQrbjmbbuZG9Wdpo8qfNUEjHATJFSycEmw6f1rGX',
+    UnderlyingAsset.PSOL,
+    AccountCoin.DEFAULT_FEATURES
+  ),
+  solToken(
+    'sol:tkmk',
+    'TOKAMAK ON SOLANA',
+    9,
+    'TKMKgSh3aADsmjr4yFWG52tkCQvmDxsQC1he1aBsi65',
+    UnderlyingAsset.TKMK,
+    AccountCoin.DEFAULT_FEATURES
+  ),
+  solToken(
+    'sol:bbsamo',
+    'BabySamoio',
+    1,
+    '7ViSurf5Ve2a8qDWFYsfU8GFmRttQvS5paJ8L94QZgo7',
+    UnderlyingAsset.BBSAMO,
+    AccountCoin.DEFAULT_FEATURES
+  ),
+  solToken(
+    'sol:csol',
+    'Solend SOL',
+    9,
+    '5h6ssFpeDeRbzsEHDbTQNH7nVGgsKrZydxdSTnLm6QdV',
+    UnderlyingAsset.CSOL,
+    AccountCoin.DEFAULT_FEATURES
+  ),
+  solToken(
```

Comment:
Does not match token reference: https://solscan.io/token/E5ndSkaB17Dm7CsD22dvcjfrYSDLCxFcMd6z8ddCk5wp

Reply:
good catch, updated and checked for others